packages/legend-studio/src/models/metada-sdlc/MetadataSdlcServiceClient.ts

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Change this file to `MetadataServerClient` and please make sure the folder name is proper - `metadata`